awx/ui_next/src/screens/Template/Survey/SurveyPreviewModal.jsx

```diff
@@ -65,7 +65,7 @@ function SurveyPreviewModal({
                       type={`survey-preview-textArea-${q.variable}`}
                       value={q.default}
                       aria-label={i18n._(t`Text Area`)}
-                      disabled
+                      isDisabled
```

Comment:
Added `isDisabled` support to TextArea https://github.com/patternfly/patternfly-react/pull/5107